src/Chapter3.hs

```diff
@@ -563,13 +583,16 @@ introducing extra newtypes.
 data Player = Player
     { playerHealth    :: Int
     , playerArmor     :: Int
-    , playerAttack    :: Int
+    , playerAttack    :: Attack
     , playerDexterity :: Int
-    , playerStrength  :: Int
+    , playerStrength  :: Strength
```

Comment:
Ideally, all these 5 fields should be separate newtypes. Newtypes add semantic meaning to fields. `Int` is just `Int` and it can mean anything. But newtypes add additional context.